module/Company/src/Company/Service/Company.php

```diff
@@ -56,12 +56,25 @@ public function getHiddenCompanyList()
      * @param mixed $package
      * @param mixed $data
      */
-    public function savePackageByData($package,$data)
+    public function savePackageByData($package, $data, $files)
     {
         $packageForm = $this->getPackageForm();
         $packageForm->setData($data);
         if ($packageForm->isValid()){
-            $package->exchangeArray($data);
+            if ($package->getType() == 'banner'){
+                $package->exchangeArray($data);
+                $file = $files['banner'];
+                try {
+                    $oldPath = $package->getImage();
+                    $newPath = $this->getFileStorageService()->storeUploadedFile($file);
+                    $package->setImage($newPath);
+                    if ($oldPath != '' && oldPath != newPath) {
+                        $this->getFileStorageService()->removeFile($oldPath);
+                    }
+                } catch (\Exception $exception) {
```

Comment:
Why an empty catch block? This is really bad practise. Also, catch statements that catch all exceptions are bad as well. Please try to catch less here.